### ISSUE TITLE
obo:CDAO_0000198 ("has_Element") unnecessarily connecting a tree to every node

### ISSUE THREAD

**Issue by gaurav** [phyloref/phyx.js#93]
We currently use obo:CDAO_0000198 ("has_Element") to connect a tree to every node. This is because of our usual JSON-LD constraint that entities not connected to one of the root element (in this case a phylogeny) will be discarded. However, there should be a cleverer way to do this.

**Comment by hlapp** [comment]
Can you do a `@reverse` for `CDAO:belongs_to_Tree`, which would at least be consistent with what would ideally just get inferred?

**Comment by gaurav** [comment]
Yes, that property is definitely more accurate! I've made that change in PR #95. I'll leave this issue open so we can fully remove these inferred properties later.

**Comment by hlapp** [comment]
I'd advocate for either closing, or re-titling the issue so it remains accurate.

Also, the main issue with expressly asserting relationships that should also be inferable is that it makes maintenance more difficult and error-prone. For example, if the axioms are changed that imply the inferred relationships, if the latter are also expressly asserted, they now need to be removed by hand. It seems this is an unlikely issue here, as one, they are software-generated anyway, and two, they are part of the tree representation(s), which we may not want to include in the released and long-term maintained clade ontology.

**Comment by gaurav** [comment]
I agree with closing, and with your reasoning for not including redundant triples.
